src/assignmenttwo/Contact.cs

```diff
@@ -0,0 +1,77 @@
+﻿// <copyright file="Contact.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Contact
+{
+    using System.Collections.Generic;
+    using System.Xml.Linq;
+
+    /// <summary>
+    /// Represents a contactsList with name, phone number, email and note.
+    /// </summary>
+    public class Contact
+    {
+        // Represents a global identifier for genarating unique values.
+        private static int id = 1;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Contact"/> class.
+        /// </summary>
+        /// <param name="name">The name of the contactsList.</param>
```

Comment:
```suggestion
        /// <param name="name">The name of the contact.</param>
```
I feel like this suggestion would help you.